src/libraries/System.Net.Http/tests/UnitTests/System.Net.Http.Unit.Tests.csproj

```diff
@@ -250,8 +250,6 @@
              Link="ProductionCode\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.Unix.cs" />
     <Compile Include="..\..\src\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.Windows.cs" Condition=" '$(TargetsWindows)' == 'true' and $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', '$(NetCoreAppCurrent)'))"
              Link="ProductionCode\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.Windows.cs" />
-    <Compile Include="..\..\src\System\Net\Http\HttpUtilities.AnyOS.cs"
```

Comment:
I think you deleted HttpUtilities.Browser.cs too, right?

Reply:
I did, but it was never included in this csproj.